packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -236,6 +240,88 @@ export class DefaultCrudRepository<T extends Entity, ID>
     );
   }
 
+  /**
+   * @deprecated
```

Comment:
Let's not add code that's already deprecated at the time of writing. Please remove this deprecated method from the pull request

Reply:
Sounds great